### PR TITLE
Add libXft-bgra to FAQ in readme

### LINKED ISSUE #227: "BadLength (poly request too large or internal Xlib length error)" when viewing files with emoji in their names
```console
$ nsxiv -v
nsxiv 28
$ nsxiv "🥰.jpg"
X Error of failed request:  BadLength (poly request too large or internal Xlib length error)
  Major opcode of failed request:  139 (RENDER)
  Minor opcode of failed request:  20 (RenderAddGlyphs)
  Serial number of failed request:  678
  Current serial number in output stream:  694
$ cp "🥰.jpg" "foo.jpg" && nsxiv foo.jpg
'🥰.jpg' -> 'foo.jpg'
$
```

### PR DESCRIPTION
**Pull request by XPhyro**
Closes #227

### CODE REVIEW

**Review by explosion-mental — APPROVED** [review]

**Comment by explosion-mental** [comment]
I _**may**_ add a link to the pull request:

[libxft](https://gitlab.freedesktop.org/xorg/lib/libxft/-/merge_requests/1)

or similar

**Comment by XPhyro** [comment]
> I _**may**_ add a link to the pull request:
> 
> [libxft](https://gitlab.freedesktop.org/xorg/lib/libxft/-/merge_requests/1)

Feel free to push to the PR. I'm not sure on how to incorporate it here.

File: README.md [hunk 1]
```diff
@@ -162,6 +162,9 @@ Yes, see [nsxiv-env](https://github.com/nsxiv/nsxiv-extra/tree/master/scripts/ns
 * Can I pipe images into nsxiv? <br>
 Yes, see [nsxiv-pipe](https://github.com/nsxiv/nsxiv-extra/tree/master/scripts/nsxiv-pipe)
 
+* nsxiv crashes when viewing images with emojis in their name. <br>
+This is an issue in libXft, install [libxft-bgra](https://github.com/uditkarode/libxft-bgra)
```
Comment (eylles) [inline README.md:166]: how about something like this?

```suggestion
This is an issue in libXft, complain on the [merge request]
(https://gitlab.freedesktop.org/xorg/lib/libxft/-/merge_requests/1) or install 
[libxft-bgra](https://github.com/uditkarode/libxft-bgra)
```

mind you i don't know how to word it better.

Comment (explosion-mental) [reply README.md:166]: I would put the link in the `issue` word

Comment (XPhyro) [reply README.md:166]: Ping @mamg22 @N-R-K

Comment (mamg22) [reply README.md:166]: eylles' suggestion sounds fine to me. Thought about using something other than "complain", but that seems a wording fit for the situation.

Comment (N-R-K) [reply README.md:166]: `either wait for libXft to merge this or install libXft-bgra instead`.

I'm not sure what's going on with the libXft situation so I'd keep neutral wording.

Comment (N-R-K) [inline README.md:166]: Hmm, shouldn't it be `issue with` rather than `issue in` ?

**Comment by XPhyro** [comment]
Any issues with the current state?

File: README.md (hunk 1 above)
Comment (eylles) [reply README.md]: from what i can gather is a rather toxic display of perfectionism, looking back not that we are innocent from that buuuut not to this extreme...

**Review by N-R-K — APPROVED** [review]

**Review by eylles — APPROVED** [review]

**Comment by rpdelaney** [comment]
Thanks everyone, this is great.

**Pull request merged**